ietf/secr/sreq/views.py

```diff
@@ -850,6 +850,8 @@ def view(request, acronym, num = None):
 
     return render(request, 'sreq/view.html', {
         'is_locked': is_locked,
```

Comment:
The template doesn't use this now, I think? If I'm right, we shouldn't pass it.

Reply:
Good catch, I remembered it appearing in other places but that must have been other templates. Will remove.